src/renderer/i18n/locales/en-US.js

```diff
@@ -468,15 +468,18 @@ export default {
       STEP2: {
         INSTRUCTIONS: {
           HEADER: 'Network selection',
-          TEXT: 'Choose the network of this profile.'
+          TEXT: 'Choose the network of this profile.',
```

Comment:
I'm not seeing this text anywhere in the stepper

Reply:
https://github.com/ArkEcosystem/desktop-wallet/blob/c8a4ef5193389b82c6ff3cd42a8cc0cb35e75977/src/renderer/pages/Profile/ProfileNew.vue#L12

Reply:
I don't see it in the UI anywhere tho 🤔 unless I overlooked it

Reply:
ah I see, it's in the left part of the screen